### PR TITLE
Build docs on appveyor

### PR DESCRIPTION
**Pull request by jenshnielsen**
This should add support for building docs with the bat file on windows and run that on appveyor

@QCoDeS/core

### CODE REVIEW

**Comment by jenshnielsen** [comment]
This is not currently building with the warnings as errors option and there seems to be some kind of issue with images

File: docs/make.bat [hunk 1]
```diff
@@ -75,13 +75,31 @@ if errorlevel 9009 (
 
 
 if "%1" == "html" (
+    :HTML
```
Comment (jenshnielsen) [inline docs/make.bat]: Indention looks wrong space vs tabs?

**Comment by WilliamHPNielsen** [comment]
Does `nbconvert` also execute the code in the notebooks when converting to `rst`? I guess not. -But that's probably why there's an `--execute` flag in there. I get it.

**Comment by jenshnielsen** [comment]
@WilliamHPNielsen Note how I got to write a goto statement

File: docs/make.bat [hunk 2]
```diff
@@ -75,13 +76,31 @@ if errorlevel 9009 (
...
+	mkdir _notebooks\DataSet\Real_instruments
+	mkdir _notebooks\driver_examples
+	mkdir _notebooks\benchmarking
+	copy _auto\qcodes.instrument_drivers.* api\generated\
+	jupyter-nbconvert --to rst "examples\*.ipynb" --output-dir=_notebooks
+	jupyter-nbconvert --to rst "examples\DataSet\Real_instruments\*.ipynb" --output-dir=_notebooks\DataSet\Real_instruments
+	jupyter-nbconvert --to rst --ExecutePreprocessor.timeout=600 --execute "examples\DataSet\*.ipynb" --output-dir=_notebooks\DataSet
+	jupyter-nbconvert --to rst "examples\driver_examples\*.ipynb" --output-dir=_notebooks\driver_examples
+	jupyter-nbconvert --to rst "examples\benchmarking\*.ipynb" --output-dir=_notebooks\benchmarking
+	goto HTML
```
Comment (WilliamHPNielsen) [inline docs/make.bat]: Shit, you're right! 😨

**Pull request merged**